src/createServer.js

```diff
@@ -1,9 +1,56 @@
 'use strict';
+/* eslint-disable no-console */
 
-function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
-}
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
+const PORT = process.env.PORT || 5701;
+const baseUrl = `http://localhost:${PORT}`;
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'text/plain');
+
+    const normalizedUrl = new url.URL(req.url, baseUrl);
+
+    const fileName = normalizedUrl.pathname.slice(6) || 'index.html';
+
+    console.log(normalizedUrl);
+    console.log(fileName);
+
+    if (!normalizedUrl.pathname.startsWith('/file')) {
+      res.statusCode = 400;
+      res.end('Route must start with /file/');
+
+      return;
+    }
```

Comment:
The check `if (!normalizedUrl.pathname.startsWith('/file'))` is redundant because the subsequent check on line 29 already verifies if the path starts with '/file/'. Consider removing this block to simplify the code.